packages/core/content-type-builder/admin/src/translations/fi.json

```
@@ -0,0 +1,208 @@
+  "form.attribute.item.text.regex": "RegExp-malli",
+  "form.attribute.item.text.regex.description": "Säännöllisen lausekkeen teksti",
+  "form.attribute.item.uniqueField": "Ainutlaatuinen kenttä",
+  "form.attribute.item.uniqueField.description": "Et voi luoda merkintää, jos olemassa on samansisältöinen merkintä",
+  "form.attribute.item.uniqueField.v5.willBeDisabled'": "Tällä hetkellä yksilölliset kentät eivät toimi oikein komponenteissa. Jos poistat tämän ominaisuuden käytöstä, kenttä poistetaan käytöstä, kunnes ongelma korjataan.",
+  "form.attribute.item.uniqueField.v5.disabled": "Tällä hetkellä yksilölliset kentät eivät toimi oikein komponenteissa. Tämä kenttä on poistettu käytöstä, kunnes se on korjattu.",
+  "form.attribute.media.allowed-types": "Valitse sallitut mediatyypit",
+  "form.attribute.media.allowed-types.option-files": "Tiedostot",
+  "form.attribute.media.allowed-types.option-images": "Kuvat",
+  "form.attribute.media.allowed-types.option-videos": "Videot",
+  "form.attribute.media.option.multiple": "Monimedia",
+  "form.attribute.media.option.multiple.description": "Paras liukusäätimille, karuselleille tai useiden tiedostojen lataamiseen",
+  "form.attribute.media.option.single": "Yksi media",
+  "form.attribute.media.option.single.description": "Paras avatarille, profiilikuvalle tai kannelle",
+  "form.attribute.settings.default": "Oletusarvo",
+  "form.attribute.text.option.long-text": "Pitkä teksti",
+  "form.attribute.text.option.long-text.description": "Paras kuvauksille, elämäkerralle. Tarkka haku ei ole käytössä.",
+  "form.attribute.text.option.short-text": "Lyhyt teksti",
+  "form.attribute.text.option.short-text.description": "Paras otsikoille, nimille, linkeille (URL). Se mahdollistaa myös tarkan haun kentällä.",
+  "form.button.add-components-to-dynamiczone": "Lisää komponentteja vyöhykkeeseen",
+  "form.button.add-field": "Lisää toinen kenttä",
+  "form.button.add-first-field-to-created-component": "Lisää ensimmäinen kenttä komponenttiin",
+  "form.button.add.field.to.collectionType": "Lisää toinen kenttä tähän kokoelmatyyppiin",
+  "form.button.add.field.to.component": "Lisää toinen kenttä tähän komponenttiin",
+  "form.button.add.field.to.contentType": "Lisää toinen kenttä tähän sisältötyyppiin",
+  "form.button.add.field.to.singleType": "Lisää toinen kenttä tähän yksittäiseen tyyppiin",
+  "form.button.cancel": "Peruuta",
+  "form.button.collection-type.description": "Paras useisiin tapauksiin, kuten artikkeleihin, tuotteisiin, kommentteihin jne.",
+  "form.button.collection-type.name": "Kokoelman tyyppi",
+  "form.button.configure-component": "Määritä komponentti",
+ "form.button.configure-view": "Määritä näkymä",
+ "form.button.select-component": "Valitse komponentti",
+ "form.button.single-type.description": "Paras yksittäiselle esiintymälle, kuten meistä, kotisivulle jne.",
+ "form.button.single-type.name": "Yksi tyyppi",
+ "from": "from",
```

```suggestion
 "from": "kohteesta",
```

A bit hard to translate without context.